git.adoc

```diff
@@ -136,8 +136,8 @@ cd to ocp docs folder, and try out the commands!
 [source,text]
 ----
 alias git-open-modified='atom $(git diff-tree HEAD HEAD~1 --no-commit-id --name-only -r)'
-alias git-push-force='git push origin "$(git rev-parse --abbrev-ref HEAD)" --force'
-alias git-commend-push-force='git commit --amend --reuse-message HEAD; git push origin "$(git rev-parse --abbrev-ref HEAD)" --force'
+alias git-push-force='git push --force-with-lease'
```

Comment:
`--force-with-lease` fails if another user created a commit on the remote branch, and you would effectively be destroying their work.

IMO, it's clunky to force-push only to the `origin` remote since this alias is useful in any repo.